README.md

```diff
@@ -1,7 +1,7 @@
 [![License: MPL v2.0](https://img.shields.io/github/license/dev-cafe/libgetkw.svg?style=for-the-badge)](https://github.com/dev-cafe/libgetkw/blob/master/LICENSE)
 
 [![Travis CI](https://img.shields.io/travis/dev-cafe/libgetkw.svg?style=for-the-badge)](https://travis-ci.org/dev-cafe/libgetkw)
-[![AppVeyor CI](https://img.shields.io/appveyor/ci/dev-cafe/libgetkw.svg?style=for-the-badge)](https://ci.appveyor.com/project/dev-cafe/libgetkw)
+[![AppVeyor CI](https://img.shields.io/appveyor/ci/dev-cafe/libgetkw.svg?style=for-the-badge)](https://ci.appveyor.com/project/bast/libgetkw)
```

Comment:
This should point to `dev-cafe`, shouldn't it?

Reply:
That's what I thought, too. But there is nothing there. It seems that Appveyor always creates projects under the name of the person who created it? It really confused me since the hook pointed to your namespace so I removed the webhook, recreated it but then it gave it my namespace.

Reply:
Probably there's something wrong in the configuration I made. Let's merge this one.